common-react/@dbeaver/ui-kit/src/Button/Button.tsx

```diff
@@ -10,6 +10,7 @@ import clsx from 'clsx';
 import './Button.css';
 import type { ControlSize } from '../types/controls.js';
 import { componentProviderWrapper } from '../componentProviderWrapper.js';
+import { Spin } from '../index.js';
```

Comment:
It's better not to import from the index (may lead to wrong circular dependencies)